boule/_ellipsoid.py

```diff
@@ -213,7 +215,28 @@ def second_eccentricity(self):
     @property
     def mean_radius(self):
         """
-        The arithmetic mean radius of the ellipsoid [Moritz1988]_.
+        The mean radius of the ellipsoid. This is equivalent to the degree 0
+        spherical harmonic coefficient of the ellipsoid shape.
+        Definition: :math:`R_0`.
```

Comment:
@MarkWieczorek would you mind adding the equation you used (the integral) as the definition of `R_0`?

Reply:
Updated. Could you check that it is ok? The equation is a little big, so I put it on a separate line.